server/src/serializers/IngredientSerializer.js

```diff
@@ -7,17 +7,27 @@ class IngredientSerializer {
       serializedIngredient[attribute] = ingredient[attribute]
     })
 
+    serializedIngredient.name = serializedIngredient.name.charAt(0).toUpperCase() + serializedIngredient.name.slice(1)
+    if (serializedIngredient.description) {
+      serializedIngredient.description = serializedIngredient.description.charAt(0).toUpperCase() + serializedIngredient.description.slice(1)
+    }
+
     return serializedIngredient
   }
 
-  static async getIngredientWithDetails(ingredient) {
+  static getIngredientWithDetails(ingredient) {
     const allowedAttributes = ["name", "id", "amount", "unit", "description"]
 
     let serializedIngredient = {}
     allowedAttributes.forEach(attribute => {
       serializedIngredient[attribute] = ingredient[attribute]
     })
 
+    serializedIngredient.name = serializedIngredient.name.charAt(0).toUpperCase() + serializedIngredient.name.slice(1)
+    if (serializedIngredient.description) {
+      serializedIngredient.description = serializedIngredient.description.charAt(0).toUpperCase() + serializedIngredient.description.slice(1)
+    }
```

Comment:
love this logic! since it appears twice across this serializer, you could define a helper method on the class to continue to DRY